content/docs/concepts/components/lifecycle/detector.md

```diff
@@ -0,0 +1,56 @@
++++
+title="Detector"
+weight=3
++++
+
+## What is the detector?
+
+Detection is the first phase of the Lifecycle. It’s done by the detector.
+In this phase, the detector is looking for an ordered group of buildpacks that will be used during the build phase.
+Detector is invoked in the build environment without any arguments (`/cnb/lifecycle/detector`) and it cannot run with root privileges.
+One of the [input files][inputs] is [`order.toml`][order] and two of its [output files][outputs] are [`group.toml`][group] and [`plan.toml`][plan].
+
+Unless some flags are passed in, the detector will use the following defaults:
+* OrderPath: `/cnb/order.toml`
```

Comment:
The reason I wrote `OrderPath` is since the environment variable is `CNB_ORDER_PATH` and I wanted it to be clear for the reader (the same comment in regards to `GroupPath` and `PlanPath`).
What do you think about changing it to `Order path` (instead of `OrderPath`) or `Path resolution for order definition`?

Reply:
I changed it a bit [here](https://github.com/buildpacks/docs/pull/277/commits/dd1ccbe664e482370ebe89f9a85d0e09cd900acf). Please let me know if you think it's clear enough now.